dear_petition/celery.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.local")
```

Comment:
Are there different settings for non-local environments we should be using?

Reply:
Yeah, it will be overwritten by Heroku config vars.